datasets/libsvm.py

```diff
@@ -1,25 +1,31 @@
-from benchopt import BaseDataset, safe_import_context
+from benchopt import BaseDataset
+
+from benchopt import safe_import_context
 
 
 with safe_import_context() as import_ctx:
-    # Dependencies of download_libsvm are scikit-learn, download and tqdm
     from libsvmdata import fetch_libsvm
 
 
 class Dataset(BaseDataset):
-    name = "news20"
-    is_sparse = True
+
+    name = "libsvm"
+
+    parameters = {
+        'dataset': ["news20.binary"],
```

Comment:
I would go for a name `news20` and have a mapping from this to `news20.binary` to keep the dataset names readable.